### PR TITLE
Adds a new ruin: the Abandoned Listening Post

### PR DESCRIPTION
**Pull request by firebudgy**
## About The Pull Request

Adds a brand new ruin to Lava Worlds, a remaster of the old listening post ruin now abandoned mid-ICW and searched by the Ramzi Clique for valuables.

Major Loot:
- Stechkin, loaded + 1 spare magazine
- Safe containing Loadsamoney spawner and Secret Documents, plus folder to hold documents in
- Blackbox 
- Telecommunications assembly, capable of receiving only
- C-20r in end room with no magazine and one spare unloaded magazine
- Scalpal, Cautery and Latex Gloves in the small infirmary

Contains for threat:
- 3 Pistol Ramzi Commandos
- 1 SMG Ramzi Commando 
- 1 Shotgun Ramzi Commando in the final room

StrongDMM Areas Screenshot:
image
StrongDMM No Areas Screenshot:
image
In-Game Screenshot:
image
## Why It's Good For The Game

With the recent Ruincut more ruins are necessary to round out the planets in-game, and replacing a poorly designed ruin from the past with a higher-quality remake is something I like doing. This also adds an interesting environment of an abandoned holding from the ICW, of which not many exist as ruins despite the ICW being such a large event. 

## Changelog

:cl: Cloudbreak
add: A new ruin, the Abandoned Listening Post.
add: Areas associated with the listening post.
fix: Single line in maps catalogue.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 1]
```diff
@@ -47,3 +47,11 @@
 	name = "\improper Mr. Deempisi portrait"
 	desc = "Under the painting a plaque reads: 'While the meat grinder may not have spared you, fear not. Not one part of you has gone to waste... You were delicious.'"
 	icon_state = "monkey_painting"
+
+/obj/structure/plaque/listeningpost
+	name = "Listening Post Commemoration Plaque"
+	engraved = TRUE
+
+/obj/structure/plaque/listeningpost/Initialize()
+	..()
```
Comment (meemofcourse) [inline code/game/objects/structures/plaques/static_plaques.dm]:
```suggestion
	. = ..()
```
i think

Comment (firebudgy) [reply code/game/objects/structures/plaques/static_plaques.dm]: thanks!

File: _maps/map_catalogue.txt [hunk 2]
```diff
@@ -90,10 +90,15 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 47)(y = 47)(z = 1)
 		Tags = "Boss Combat Challenge", "Major Loot", "Shelter", "Lava"
 
-		File Name = "_maps\RandomRuins\BeachRuins\lavaland_crashed_starwalker.dmm"
+		File Name = "_maps\RandomRuins\LavaRuins\lavaland_crashed_starwalker.dmm"
```
Comment (Latentish) [inline _maps/map_catalogue.txt:93]: I'm going to let this slip through because i'm epic and hilarious.

**Review by Latentish — DISMISSED** [review]
No suicide implications please.
image

Don't use RTGs as ruinloot.
image

image
Couple things. Gas tanks don't connect like this, it needs a pump in front of it. This room is all plating. Use a mix of tiles and catwalks to go over important pipes and such.

image
Two things. One, this looks rather strange because these cement tiles are supposed to only be used if paired with other cement tiles. Two, these tiles need to have the same initialization atmos as the rest of its environment or it'll cause an active turf on load, which means it takes longer and takes up resources. If you cannot find a version of the tile you want to use that matches the planetary atmos, you'll need to make a subtype of it.

image
All of these places need more clutter and decoration.

image
These areas need more decal and floorwork.

image
This air cycler needs a controller.

You need to add lightswitches.

Overall this ruin isn't bad, with a good handful of touchups this would make a good minor ruin. You can get help with addressing this review in ruinyard.

File: _maps/map_catalogue.txt (hunk 2 above)
Comment (firebudgy) [reply _maps/map_catalogue.txt:93]: I believe most if not all of these concerns have been addressed now, I've updated my PR's images to reflect the new version of the ruin.

**Review by Latentish — CHANGES_REQUESTED** [review]
Add a guy to the engine room and you get  a merge. God work here.

**Review by Latentish — APPROVED** [review]

**Pull request merged**